js/script1.js

```diff
@@ -0,0 +1,226 @@
+let game_started = false;
+let matches = 0;
+let moves = 0;
+let openedCardIds = []
```

Comment:
Is the recentlyOpenedCards or allOpenedCards ?
Variable naming is very critical for effective & faster review, faster delivery & code maintenance